src/lib/tour-overlay/TourOverlay.component.tsx

```diff
@@ -48,14 +49,14 @@ export const TourOverlay = React.forwardRef<TourOverlayRef, TourOverlayProps>((p
   const useNativeDriver = useMemo(() => Platform.select({
     android: false,
     default: false,
-    ios: true
+    ios: false
```

Comment:
Is is needed to specify the Native Driver for each OS if all of them are false?

Reply:
I suppose that it is ok to have it like this if we want to then give a have to opt-in/opt-out. A comment would good to have explaining the change and the steps taken to support the Native Driver.

Reply:
Yeah, for now it's false on both, but later users will be able to choose for themselves. I'll leave a comment here too. Thanks! 🙂